src/AutoRest.CSharp/Common/Generation/Writers/SerializationWriter.cs

```diff
@@ -56,6 +54,9 @@ private void WriteObjectSerialization(CodeWriter writer, SerializableObjectType
 
             using (writer.Namespace(declaration.Namespace))
             {
+                //[TODO] Azure.Core is not needed to compile, but previous version of writer has been adding it.
+                writer.UseNamespace("Azure.Core");
```

Comment:
we keep this in order to reduce the amount of diff in the regen?

Reply:
well in the regen, I saw that we are actually adding `Azure.Core` namespace in quite a few test projects.
Maybe we could just remove this.

Reply:
There are much more files where `Azure.Core` will be removed. I will remove this line in separate commit to this PR once everything else is finalized.